### PR TITLE
Add Java to the getting started section

### PR DESCRIPTION
**Pull request by markwallace-microsoft**
### Motivation and Context

Please help reviewers and future users, providing the following information:
  1. Why is this change required? Update the readme to include getting started information for the Java version.

### Description
Preview the file to see the changes.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#dev-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

**Review by johnoliver — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ Semantic Kernel is available to explore AI and build apps with C# and Python:
           <a href="python/README.md">Using Semantic Kernel in Python</a>
         </div>
       </td>
+      <td>
+        <img align="left" width=52px src="https://raw.githubusercontent.com/devicons/devicon/master/icons/java/java-original.svg">
```
Comment (brunoborges) [inline README.md:63]: This is an old Java logo.

This is the new one: https://en.wikipedia.org/wiki/Java_%28programming_language%29#/media/File:Java_programming_language_logo.svg

File: README.md [hunk 2]
```diff
@@ -67,7 +73,7 @@ See the [Feature Matrix](https://learn.microsoft.com/en-us/semantic-kernel/get-s
 
 The quickest way to get started with the basics is to get an API key
 (OpenAI or Azure OpenAI)
-and to run one of the C# or Python console applications/scripts:
+and to run one of the C#, Python console applications/scripts:
```
Comment (brunoborges) [inline README.md:76]: C#, Python, **and Java** console applications/scripts.

**Review by brunoborges — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (markwallace-microsoft) [reply README.md:63]: @brunoborges says that one is not free to use: https://en.wikipedia.org/wiki/File:Java_programming_language_logo.svg

The one I'm using came from the same location as the other icons we are using.

**Review by johnoliver — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (brunoborges) [reply README.md:63]: Please update to the new logo.

**Review by glahaye — APPROVED** [review]

**Review by johnoliver — APPROVED** [review]

**Review by brunoborges — APPROVED** [review]
LGTM

**Pull request merged**